client/src/components/new-request/analyze-pending.vue

```diff
@@ -1,17 +1,17 @@
 <template>
   <MainContainer id="analyze-pending-container">
     <template v-slot:container-header>
-      <v-col cols="auto h4">
+      <v-col cols="auto font-weight-bold h6 ml-3">
         You are searching for a name for a
         {{ entityText === ' BC Corporation' && location.text === ' BC' ? '' : ' ' + location.text }} {{ entityText }}
       </v-col>
     </template>
     <template v-slot:content>
-      <v-row no-gutters justify="space-between" align-content="space-around">
-        <NameInput class="mt-3" is-search-again="true"/>
+      <v-row no-gutters>
+        <NameInput is-search-again="true"/>
       </v-row>
       <v-row justify="center" class="mt-n7 mb-n3 py-0">
-        <v-col cols="12" class="text-center">
+        <v-col cols="12" class="text-center copy-small">
```

Comment:
Would text-body-2 work here instead of copy-small?

Reply:
I'm not sure of the differences, will look!

Reply:
Absolutely, no issue! Updated.

Reply:
Yeah, I'm trying to get to a limited subset of global styles (for consistency!). Thanks for this change.